ui/raidboss/data/06-ew/trial/endsinger.txt

```diff
@@ -0,0 +1,100 @@
+# Endsinger Normal
+# -p "662C:13.7" "6B58:1000" "6B5B:2000"
+# -it 'The Endsinger' 'Kakodaimon'
+# -oc 'The Endsinger' 'Kakodaimon'
+# -ii 6281 6C61 6660 6661 6C6A 664B 6653 6643 67FA 6642 6648 67FB 6655
+
+# Not sure if/where P1 loops
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# P1
+0.0 "--sync--" sync /Engage!/ window 0,1
+13.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/ window 15,15
```

Comment:
```suggestion
8.7 "--sync--" sync / 14:[^:]*:The Endsinger:662C:/ window 15,15
13.7 "Elegeia" sync / 1[56]:[^:]*:The Endsinger:662C:/
```

If there's no zone closing, it's nice to sync to the starts casting so the timeline starts sooner.